[UNOMI-638] - Updates documentation in preparation for Unomi 2 (#472)
apache/unomi: manual/src/main/asciidoc/whats-new.adoc

//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
=== What's new in Apache Unomi 2.0

Apache Unomi 2 is a new release focused on improving core functionalities and robustness of the product. 

> The introduction of tighter data validation with JSON Schemas required some changes in the product data model, which in turn presented an opportunity for noticeable improvements of the overall performances.

Reviewer: ```suggestion
The introduction of tighter data validation with JSON Schemas required some changes in the product data model, which presented an opportunity for noticeable improvements in the overall performance.
```

> This new release also introduces a first (beta) version of Unomi GraphQL API.

Reviewer: ```suggestion
This new release also introduces a first (beta) version of the Unomi GraphQL API.
```

> ==== Scopes are required

Reviewer: ```suggestion
==== Scopes declarations are now required
```

> Scopes are required in Unomi 2. When submitting an event and specifying a scope, 

Reviewer: ```suggestion
Scopes declarations are now required in Unomi 2. When submitting an event and specifying a scope, 
```
